src/oidcop/utils.py

```diff
@@ -19,7 +19,7 @@ def load_yaml_config(file_name):
     return c
 
 
-def yaml_to_py_stream(file_name):
+def yaml_to_py_stream(file_name): # pragma: no cover
```

Comment:
all these utilities are not used in the project, so I'm ok to keep them for future needs but marked as uncovered to give them a minor relevance in the code

Reply:
I'm actually OK with weeding them out.